modules/light-clients/08-wasm/types/vm.go

```diff
@@ -114,7 +114,10 @@ func wasmInstantiate(ctx sdk.Context, cdc codec.BinaryCodec, clientStore storety
 	checksum := cs.Checksum
 	resp, err := instantiateContract(ctx, clientStore, checksum, encodedData)
```

Comment:
This variable name "resp" is misleading now since the variable contains a result (i.e. success or error struct) and response only referred to the success case

Reply:
Good point. I can change the name.